README.md

```diff
@@ -47,15 +90,17 @@ function WeekSalesChart() {
 }
 ```
 
-This approach makes the component simpler and more efficient, as it only displays the data you provide. You can still customize the component using your own styles.
+You can customize the component using your own styles.
 
 <p align="center">
-  <img src="https://storybook.propeldata.com/images/time-series.png">
+  <img src="https://storybook.propeldata.com/images/time-series.png"/>
 </p>
 
-### Connected Mode: The Power of Propel at one import away
+### Connected Mode: The Power of Propel one import away
```

Comment:
Recommend introducing this first, before "Static" mode. It's what ultimately matters, IMO. Also, recommend losing the subheading?

```suggestion
### Connected Mode
```